src/inverseRobot.js

```diff
@@ -23,8 +23,18 @@
  *
  * @return {object}
  */
-function inverseRobot(robot) {
-  // write code here
-}
+
+const inverseRobot = (robot) => {
+  const robotReverse = {};
+
+  for (const key of Object.keys(robot)) {
```

Comment:
```suggestion
  for (const key in robot) {
```
You could use `for`-`in` loop here